src/helpers/colors.js

```diff
@@ -0,0 +1,10 @@
+export const color = ["#E3F2FD",
```

Comment:
I'd prefer avoiding hex codes, but we can leave this be for now until we finalize the colors